core/genesis.go

```diff
@@ -475,24 +495,34 @@ func (g *Genesis) IsVerkle() bool {
 	return g.Config.IsVerkle(new(big.Int).SetUint64(g.Number), g.Timestamp)
 }
 
+// IsIsthmus indicates whether Isthmus is active at genesis time.
+func (g *Genesis) IsIsthmus() bool {
+	return g.Config.IsIsthmus(g.Timestamp)
+}
+
```

Comment:
IMHO we should not add additional public-API to the `Genesis` struct. It's a one-liner anyway, not worth the diff.